contracts/Lens/PoolLens.sol

```diff
@@ -302,4 +302,179 @@ contract PoolLens is ExponentialNoError {
                 underlyingPrice: priceOracle.getUnderlyingPrice(address(vToken))
             });
     }
+
+    /**
+     * @dev Struct for Pending Rewards for per market
+     */
+    struct PendingReward {
+        address vTokenAddress;
+        uint256 amount;
+    }
+
+    /**
+     * @dev Struct for Reward of a single reward token.
+     */
+    struct Reward {
```

Comment:
The name of the struct is singular but the values in contains are plural. `PoolRewards` is a more descriptive name for example